examples/neo4j/README.md

```diff
@@ -0,0 +1,28 @@
+# Neo4j example
+
+Basic tinyhttp example using [neo4j](https://neo4j.com/docs/).
```

Comment:
I'd recommend to change it "A simple app using tinyhttp and neo4j"

Reply:
done

Reply:
done